apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3556,11 +3553,27 @@ int tc_filesystem_main(int argc, char *argv[])
 
 	tc_fs_vfs_rename();
 	tc_fs_vfs_ioctl();
+
+#ifndef CONFIG_FS_AUTOMOUNT_PROCFS
```

Comment:
I think this condition is needed in tc_fs_procfs_main instead of mouting of procfs here.

Reply:
I know that you changed these because of stat of procfs in tc_fs_vfs_stat().
I think it is better to keep tc_fs_vfs_stat() and add condition about procfs to "ret = stat(PROCFS_PATH, &st);" in tc_fs_vfs_stat.